### ISSUE TITLE
GLPI Invetory Net Discovery won't start during the selected schedule.

### ISSUE THREAD

**Issue by Maqsence** [glpi-project/glpi-inventory-plugin#449]
### Describe the bug

Net Discovery jobs don't automatically start within the selected time gap.

It works well if I start it manually with the force button.

### To reproduce

Create a net discovery job (link an agent, an ip range... and a time schedule)

The jobs will never start during the selected period.

### Expected behavior

The job starting during the schedule.

### Operating system

Windows

### GLPI Agent version

1.5, Other (See additional context below)

### GLPI version

10.0.10

### GLPIInventory plugin

1.3.0

### Additional context

Tried with agent 1.6.1 today, does not seem to work either.

**Comment by g-bougard** [comment]
Hi @Maqsence 

I confirm the agent needs to be run as a service at least during the required wakeup period. You may still want to configure a start and a stop of the service around the required wakeup period if you really don't want it to run all the time. Also you may not use the wakeup automatic action but schedule a run with the `--force` option during that period. In that last case, you still don't really need to have the agent running as a service, you just manage to force the task during the expected period.

If you continue with the service mode, you'll also have to check in the agent log if it really receives `/now` requests on its httpd interface from the glpi server. Also `no-httpd` option MUST NOT be set to 1.

**Comment by maxwal41** [comment]
Hello.
I also observed such a problem and was waiting for a solution to appear in this topic. I use fusion as an agent, which runs in the scheduler once an hour.
This problem is very similar to when the mail receiver stops receiving incoming mail. It happens to me if I shut down or restarted the web server.
To solve it, you need to manually complete all pending tasks in the "Automatic actions" section. After that, the mail begins to be accepted and the status of the automatic inventory or detection changes to "prepared".
In my case, this works even if the agent is not used as a service.

**Comment by stonebuzz** [comment]
Please try with latest version of glpiinventory plugin

**Comment by trasher** [comment]
No feedback, I close

**Comment by Maqsence** [comment]
Hi,

Sorry for the lack of answers. 

I get this error in the logs : 

`[Wed Feb 28 05:26:55 2024][error] #03361, [http client] communication error: 500 Internal Server Error, Unable to create item.
[Wed Feb 28 05:27:16 2024][error] #03448, [http client] communication error: 500 Internal Server Error, Unable to create item.`

Then, in glpiinventory, i have : 

![image](https://github.com/glpi-project/glpi-inventory-plugin/assets/151037061/440d03b9-5e4a-4933-82ba-06717c5c7b08)

Basically, critical error.

Any ideas ?

Thanks for your time

**Comment by cedric-anne** [comment]
Hi,

You should try to manually import the inventory file generated by the agent. Most of the time, this error is due to unicity rules that blocks the creation of the item.
